src/content/contributing/release-process/_index.en.md

```diff
@@ -67,7 +71,69 @@ Verify that the build successfully completes as indicated by a green checkmark a
 > https://quay.io/repository/submariner/lighthouse-controller?tab=tags
 > https://quay.io/repository/submariner/lighthouse-coredns?tab=tags
 
-<!-- TODO: Aswin: the openshift-coredns part -->
+## Build Lighthouse CoreDNS
+
+1) Get the coreDNS and change to the folder
+
+```bash
+go get github.com/openshift/coredns
+cd $GOPATH/src/github.com/openshift/coredns
+```
+2) Build the image and push
+
+```bash
+COREDNS_VERSION=<VERSION-NO>
+LH_COREDNS_VERSION=<LH-VERSION-NO>
+COREDNS_IMAGE="lighthouse-coredns:${LH_COREDNS_VERSION}"
+git checkout -b ${COREDNS_VERSION}
+sed -i '/^kubernetes:kubernetes/a lighthouse:github.com/submariner-io/lighthouse/plugin/lighthouse' plugin.cfg
+sed -i '/^github.com/aws/aws-sdk-go/a github.com/submariner-io/lighthouse v0.2.0' go.mod
+sed -i '$a replace\ k8s.io\/apimachinery\ =>\ k8s.io\/apimachinery\ v0.0.0-20190313205120-d7deff9243b1' go.mod
+sed -i '$a replace\ github.com\/openzipkin-contrib\/zipkin-go-opentracing\ =>\ github.com\/openzipkin-contrib\/zipkin-go-opentracing\ v0.3.5' go.mod
+docker build  -f Dockerfile.openshift -t quay.io/submariner/${COREDNS_IMAGE} .
+docker push quay.io/submariner/${COREDNS_IMAGE}
+```
+
+## Build Lighthouse Cluster-DNS-Operator
+
+1) Clone  the Lighthouse Cluster-DNS branch and create a version branch
+
+```bash
+git clone https://github.com/submariner-io/cluster-dns-operator.git
+git checkout -b <VERSION-NO>
+```
+
+2) Change the coredns and cluster dns operator versions.
+
+```
+	- name: dns-operator
+          terminationMessagePolicy: FallbackToLogsOnError
+          image: quay.io/submariner/lighthouse-cluster-dns-operator:<VERSION-NO>
+          command:
+          - dns-operator
+          env:
+          - name: RELEASE_VERSION
+            value: "0.0.1-snapshot"
+          - name: IMAGE
+            value: quay.io/submariner/lighthouse-coredns:<VERSION-NO>
+```
+
+For example,
+https://github.com/submariner-io/cluster-dns-operator/pull/1/files#diff-8c85a5683e17f9599cbfc641cceaa040R33
```

Comment:
nit - that PR actually changed the image location/name as well.  However I think it's straightforward enough not to need to look at a PR.

Reply:
I think it's useful in case there is some confusion, please let's keep it for now until we can automate it.